### ISSUE TITLE
document how to give options to adapter

### ISSUE THREAD

**Issue by alexanderadam** [lostisland/faraday#572]
I'm using the patron adapter and occasionally running into the `Maximum (5) redirects followed` exception.
Patron allows to configure it with `max_redirects` but I'm can't find how to configure it with faraday. :cry:

**Comment by iMacTia** [comment]
Hi @alexanderadam,

I'm in the process of adding the feature you need with #651 
Please have a look at the PR for additional information 😃

**Comment by iMacTia** [comment]
#651 released with v0.11.0
This is the usage in your case:

```ruby
conn = Faraday.new(...) do |f|
  # Insert more middlewares here  
  f.adapter :patron do |session| # yields Patron::Session
    session.max_redirects = 10
  end
end
```
